Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -21,9 +21,15 @@ import SwiftProtobuf
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 public final class ReflectionService: CallHandlerProvider, Sendable {
-  private let reflectionService: ReflectionServiceProvider
+  private let reflectionServiceProvider: ProviderType
```

Comment:
This naming doesn't need to be so verbose: we're within the `ReflectionService` so we can just use "provider".